transport/wseb/src/main/java/org/kaazing/gateway/transport/wseb/WsebSession.java

```diff
@@ -179,6 +179,14 @@ public WsebSession(int ioLayer,
         transportSession.setHandler(transportHandler);
     }
 
+    protected void setIoAlignment0(Thread ioThread, Executor ioExecutor) {
+        transportSession.setIoAlignment(ioThread, ioExecutor);
+        super.setIoAlignment0(ioThread, ioExecutor);
+
+        // TODO shoud we detachWriter() too so that newly attached writer gets aligned by default ?
```

Comment:
Actually given the writer is the parent we don't need to do this. I do wonder a bit about the reader.